__tests__/integrations/python.Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 
 # Build runtime image
 # TODO add this to base.Dockerfile?
-FROM node:18
+FROM node:16
```

Comment:
This is the only CI Docker workflow that was using Node 18 and because I'm pulling in `isomorphic-fetch` it's running `fetch` calls with Undici. Unsurprisingly it's causing inconsistencies between tests running on Node 16.